lib/rules/require-unicode-regexp.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
         type: "suggestion",
 
         docs: {
-            description: "Enforce the use of `u` flag on RegExp",
+            description: "Enforce the use of `u` and `v` flag on RegExp",
```

Comment:
```suggestion
            description: "Enforce the use of `u` or `v` flag on RegExp",
```

I think "or" is correct here.